### ISSUE TITLE
a11y: va-backtop

### ISSUE THREAD

**Issue by kushich** [epicmaxco/vuestic-ui#3267]
- [x] step 1: investigate a11y guides for this component;
- [ ] step 2: discuss with team;
- [ ] step 3: implement required functionality;
- [ ] step 4: add docs page section about a11y.

**Comment by kushich** [comment]
- [ ] It should be the last element that receives focus on the page.
- [x] Also it should have `aria-label`
